src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = {};
+  const VALUES = Object.values(robot);
+  const KEYS = Object.keys(robot);
+  const REMOVE_DUBLICATE_VALUES = new Set(VALUES);
+
+  if ([...REMOVE_DUBLICATE_VALUES].length !== VALUES.length) {
```

Comment:
I think it would be better to use the object function to define the repeating field. Especially considering the fact that we were given a hint in the checklist. [Cheklist - paragraph 2].
`if (robot.hasOwnProperty(key))`
But I`m not sure about this, u can re push your project and mentors will say their opinion.

Reply:
Good point